groups/cvo2/locals.tf

```diff
@@ -2,12 +2,13 @@
 # Locals
 # ------------------------------------------------------------------------
 locals {
-  account_ids           = data.vault_generic_secret.account_ids.data
-  admin_cidrs           = values(data.vault_generic_secret.internal_cidrs.data)
-  netapp_account_data   = data.vault_generic_secret.netapp_new_account.data
-  netapp_connector_data = data.vault_generic_secret.netapp_new_connector.data
-  netapp_cvo_data       = data.vault_generic_secret.netapp_new_cvo.data
-  sg_id                 = data.vault_generic_secret.existing_sg.data
+  account_ids            = data.vault_generic_secret.account_ids.data
+  admin_cidrs            = values(data.vault_generic_secret.internal_cidrs.data)
+  netapp_account_data    = data.vault_generic_secret.netapp_new_account.data
+  netapp_connector_data  = data.vault_generic_secret.netapp_new_connector.data
+  netapp_cvo_data        = data.vault_generic_secret.netapp_new_cvo.data
+  sg_mediator_data       = data.vault_generic_secret.existing_sg.data
+  sg_mediator_name       = local.sg_mediator_data["security_group_name"]
```

Comment:
For consistency, it would be best to slightly rename these to be `mediator_sg_*`